src/transaction/boot_cl.c

```diff
@@ -5653,7 +5653,14 @@ boot_define_view_index (void)
 	  "CASE [i].[is_reverse] WHEN 0 THEN 'NO' ELSE 'YES' END AS [is_reverse], "
 	  "[i].[class_of].[class_name] AS [class_name], "
 	  "CAST ([i].[class_of].[owner].[name] AS VARCHAR(255)) AS [owner_name], " /* string -> varchar(255) */
+#if defined(SUPPORT_DEDUPLICATE_KEY_MODE)
+          "NVL2((SELECT 1 FROM _db_index_key [k] WHERE [k].index_of.class_of = [i].class_of "
+                                                   "AND [k].index_of.index_name = [i].[index_name] "
+                                                   "AND [k].key_attr_name LIKE " DEDUPLICATE_KEY_ATTR_NAME_LIKE_PATTERN
+                 "), ([i].[key_count] - 1), [i].[key_count]) AS [key_count], "
+#else
 	  "[i].[key_count] AS [key_count], "
+#endif          
```

Comment:
How about changing like below?
```suggestion
#if defined(SUPPORT_DEDUPLICATE_KEY_MODE)
	  "NVL2 ("
	      "("
		"SELECT 1 "
		"FROM "
		  /* CT_INDEXKEY_NAME */
		  "[%s] [k] "
		"WHERE "
		  "[k].index_of.class_of = [i].class_of "
		  "AND [k].index_of.index_name = [i].[index_name] "
		  "AND [k].key_attr_name LIKE " DEDUPLICATE_KEY_ATTR_NAME_LIKE_PATTERN
	      "), "
	      "([i].[key_count] - 1), "
	      "[i].[key_count]"
	    ") AS [key_count], "
#else
	  "[i].[key_count] AS [key_count], "
#endif
```

The code below must be added when naming the system table.
```
#if defined(SUPPORT_DEDUPLICATE_KEY_MODE)
	CT_INDEXKEY_NAME,
#endif
	CT_INDEX_NAME,
	AU_USER_CLASS_NAME,
	AU_USER_CLASS_NAME,
	CT_CLASSAUTH_NAME,
	AU_USER_CLASS_NAME);
```

Reply:
OK,